naptha_sdk/client/hub.py

```diff
@@ -300,6 +313,9 @@ async def user_setup_flow(hub_url, public_key):
             case dict(), True, True, False:
                 # User exists, attempt to sign in
                 logger.info("Using user credentials in .env. User exists. Attempting to sign in...")
+                if os.getenv("PRIVATE_KEY") and is_hex(os.getenv("PRIVATE_KEY")):
+                    write_private_key_to_file(os.getenv("PRIVATE_KEY"), username)
```

Comment:
@Satti-Gowtham if PRIVATE_KEY is set, doesn't that mean the private key is already written to file?